src/maintenance/maintenanceInteractor.spec.ts

```diff
@@ -17,6 +36,20 @@ describe('get maintenance status', () => {
         await expect(maintenance.getMaintenanceStatus())
             .resolves.toBe(true);
     });
+    it('should return undefined because the user is an admin', () => {
+      expect(maintenance.setAuthorization(AUTHORIZEDMOCKUSER)).toBeUndefined();
```

Comment:
Instead of checking for undefined, let's check to make sure that the function didn't throw an error. Sometimes tests for undefined lead to misleading results (false positives). 
```suggestion
      expect(maintenance.setAuthorization(AUTHORIZEDMOCKUSER)).not.toThrowError();
```

Reply:
I changed this not.toBe(resource error). Since the function will only ever throw the resource error for invalid access if the token is undefined or the access group doesn't have admin in it I tested to see if the value returned isn't the error.